apps/emqx_management/src/emqx_mgmt.erl

```diff
@@ -461,17 +469,34 @@ set_keepalive(_ClientId, _Interval) ->
 
 %% @private
 call_client(ClientId, Req) ->
-    Results = [call_client(Node, ClientId, Req) || Node <- emqx:running_nodes()],
-    Expected = lists:filter(
+    case emqx_cm_registry:is_enabled() of
+        true ->
+            do_call_client(ClientId, Req);
+        false ->
+            call_client_on_all_nodes(ClientId, Req)
+    end.
+
+call_client_on_all_nodes(ClientId, Req) ->
+    Nodes = emqx:running_nodes(),
+    Results = call_client(Nodes, ClientId, Req),
+    {Expected, Errs} = lists:foldr(
```

Comment:
nit: foldr changes the call order comparing to before this refactoring ?

Reply:
I've replaced serial call `[call_client(Node, ClientId, Req) || Node <- emqx:running_nodes()]` with `erpc:multicall` (in management_proto), so the call order must be the same (`erpc:multicall` sends requests from left to right). 
foldr is used to preserve order of the results in Acc.